src/Training.AirBnb.Clone.Frontend/src/App.vue

```diff
@@ -1,10 +1,32 @@
 <template>
 
   <!--    <main-logo/>-->
-  <home-view/>
+  <home-view @sign-in-up-request="toggleModal"/>
+
+  <sign-in-up-modal :modal-active="modalActive" @close-modal="toggleModal"/>
```

Comment:
As we are teleporting `SignInUpModal` to body, why not add it to `MainHeader` component ?